jmix-superset/superset-flowui/src/main/resources/ui.xsd

```diff
@@ -0,0 +1,55 @@
+<!--
+  ~ Copyright 2024 Haulmont.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<xs:schema targetNamespace="http://jmix.io/schema/superset/ui"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:layout="http://jmix.io/schema/flowui/layout"
+           xmlns="http://jmix.io/schema/superset/ui"
+           elementFormDefault="qualified">
+    <xs:element name="dashboard">
+        <xs:complexType>
+            <xs:complexContent>
+                <xs:extension base="layout:baseComponent">
+                    <xs:sequence minOccurs="0">
+                        <xs:element name="datasetConstraints" type="datasetConstraintsType"/>
+                    </xs:sequence>
+
+                    <xs:attribute name="embeddedId" type="xs:string"/>
+                    <xs:attribute name="url" type="xs:string"/>
+                    <xs:attribute name="titleVisible" type="xs:boolean"/>
+                    <xs:attribute name="chartControlsVisible" type="xs:boolean"/>
+                    <xs:attribute name="filtersExpanded" type="xs:boolean"/>
+
+                    <xs:attribute name="datasetConstraintsProviderBean" type="xs:string"/>
+                    <xs:attribute name="guestTokenProviderBean" type="xs:string"/>
```

Comment:
I have a doubt about these properties. Do you think it will be convenient to type bean names here? What are the cases users will do that? Maybe we might just provide setters in the SupersetDashboard component that will set the token provider is someone really want to change this behavior? The same for constraints provider.

Reply:
Probably, we can remove the `guestTokenProviderBean` property from the schema, since it is a rare case when we need to change the guest token provider only for one component. I think the main case will be overriding the `DefaultGuestTokenProvider` for all components in the system. I would leave the `datasetConstraintsProviderBean` in order to completely configure the component in XML without Java code. However, I didn't find similar cases in Jmix and add-ons. So, I'll remove these properties because it is easier to add an XML property than to remove it.

Reply:
Fixed